src/main/java/org/lobid/resources/RdfModel2ElasticsearchEtikettJsonLd.java

```diff
@@ -87,33 +87,46 @@ public RdfModel2ElasticsearchEtikettJsonLd(final Object jsonLdContext) {
 	@Override
 	public void process(final Model originModel) {
 		mainNodeId = null;
-		splitModel2ItemAndResourceModel(originModel);
+		extractItemFromResourceModel(originModel);
 	}
 
-	private void splitModel2ItemAndResourceModel(final Model originalModel) {
+	private void extractItemFromResourceModel(final Model originalModel) {
 		Model copyOfOriginalModel =
 				ModelFactory.createModelForGraph(originalModel.getGraph());
 		final ResIterator subjectsIterator = originalModel.listSubjects();
 		// iterate through all nodes
 		while (subjectsIterator.hasNext()) {
 			final Resource subjectResource = subjectsIterator.next();
 			Model submodel = ModelFactory.createDefaultModel();
-			// only extract sub nodes we don't want to keep in the main model
+			// extract sub nodes (items) => will become documents on their own
 			if (subjectResource.isURIResource()) {
 				if (subjectResource.getURI().startsWith(LOBID_ITEM_URI_PREFIX)) {
 					submodel = extractSubmodel(submodel, subjectResource);
 					toJson(submodel,
 							subjectResource.getURI().toString().replaceAll("#!$", ""));
-					// remove the newly created sub model from the main node
-					copyOfOriginalModel.remove(submodel);
+					// remove some properties from that item which clings to the main node
+					removeProperty(copyOfOriginalModel, subjectResource, "itemOf");
+					removeProperty(copyOfOriginalModel, subjectResource, "describedby");
```

Comment:
I'm a bit confused, since `describedby` is now `describedBy` (see https://github.com/hbz/lobid-resources/issues/358), yet this seems to work, as there is no `describedBy` in the embedded items. Is it still `describedby` internally somehow?